Expensify/App
# Make unread indicator for web favicon larger

This updates the favicon for web when there are unread messages.

### Details


### Fixed Issues
https://github.com/Expensify/Expensify.cash/issues/2658

### Tests
1. Open up Expensify.cash for web, and make sure the favicon when there is unread messages has been updated. The red dot on the icon should be larger. 

### QA Steps
1. Open up Expensify.cash for web, and make sure the favicon when there is unread messages has been updated. The red dot on the icon should be larger. 

### Tested On


### Screenshots
This change is for web only. 

#### Web
No Unreads | Unreads
--------
![image](https://user-images.githubusercontent.com/2319350/118516635-58e67480-b6eb-11eb-98c7-226892960bf8.png) | ![image](https://user-images.githubusercontent.com/2319350/118516899-9814c580-b6eb-11eb-8ce7-105c6374d700.png)

#### Mobile Web
N/A

#### Desktop
N/A

#### iOS
N/A

#### Android
N/A
